hw_3/Task1.js

```diff
@@ -0,0 +1,78 @@
+// Task 1. Работа с переменными:
+
+"use strict";
+
+/* 
+1. Создать переменную “item_1”
+2. Присвоить переменной item_1 цифру 5.
+3. Вывести в консоль item_1.
+*/
+
+const item_1 = 5;
+console.log (item_1);
+
+/* 
+4. Создать переменную “item_2”
+5. Присвоить переменной item_2 цифру 3.
+6. Вывести в консоль item_2.
+*/
+
+const item_2 = 3;
+console.log (item_2);
+
+/*
+7. Создать переменную “item_3”
+8. Присвоить переменной item_3 сложение item_1 и item_2.
+9. Вывести в консоль item_3.
+*/
+
+const item_3 = item_1 + item_2;
+console.log (item_3);
+
+/*
+10. Создать переменную “item_4”
+11. Присвоить переменной item_4 строку “Yolochka”
+12. Вывести в консоль item_4.
+*/
+
+const item_4 = 'Yolochka';
+console.log (item_4);
+
+//13. Вывести в консоль сложение item_3 и item_4.
+
+console.log (item_3 + item_4);
+
+// 14. Вывести в консоль умножение item_3 и item_4.
+
+console.log (item_3 * item_4);
+
+/* 
+15. Создать переменную “item_5”
+16. Присвоить переменной item_5 переменную item_3
+*/
+
+const item_5 = item_3;
+
+/*
+17. Создать переменную item_6.
+18. Создать переменную item_6_type
+19. Присвоить переменной item_6 значение 15
+20. Присвоить переменной item_6_type тип переменной item_6
+21. Вывести в консоль тип данных item_6 в виде ——  “item_6 == ”  item_6,  “item_6_type == ”  item_6_type ——  
+*/
+
+const item_6 = 15;
+const item_6_type = typeof item_6;
+console.log('item_6 == ' + typeof item_6 + ', item_6_type == ' + typeof item_6_type);
+
+/* 
+22. Создать переменную item_7 и в ней преобразовать item_6 в String.
+23. Создать переменную item_7_type
+24. Присвоить переменной item_7_type тип переменной item_7
+25. Вывести в консоль тип данных item_7 в виде ——  “item_7 == ”  item_7,  “item_7_type == ”  item_7_type ——  
+*/
+
+const item_7 = 'item_6';
```

Comment:
Pay attention to the task description - need to convert `item_6` value to string, not assign `string "item_6" to item_7`

Reply:
Done